src/store/wallet/effects/send/send.ts

```diff
@@ -677,6 +686,17 @@ export const publishAndSign =
   async (dispatch, getState) => {
     return new Promise(async (resolve, reject) => {
       let password;
+      const {
+        APP: {biometricLockActive},
+      } = getState();
+
+      if (biometricLockActive) {
```

Comment:
Small thing I noticed, if I try to send it will create a txp, but if I cancel the biometric on purpose, it fails to delete the txp. We should probably move this check as early as possible to avoid side effects.